lib/sgn-sdk.js

```diff
@@ -71,8 +73,19 @@ if (isBrowser()) {
         if (trackId) {
             config.eventTracker = new SGN.EventsKit.Tracker({trackId});
         }
+        if (businessId) {
+            config.businessId = businessId;
+        }
 
         SGN.config.set(config);
+
+        if (component === 'view-publication') {
+            SGN.CoreUIKit.PagedPublication(scriptEl).render();
```

Comment:
We need to pave the way for Incito too. So perhaps component should be view-paged-publication OR the SDK handles which one to use automatically.